lib/Aftok/Currency/Zcash.hs

```diff
@@ -137,7 +144,7 @@ rpcValidateZAddr mgr cfg addr = runExceptT $ do
         Nothing -> Left (RPCError DataMissing)
         Just Z.Sprout -> Left (RPCError SproutAddress)
         Just Z.Sapling -> Right (Z.Address addr)
-      else Left $ RPCError ZAddrInvalid
+      else Left (RPCError ZAddrInvalid)
```

Comment:
I'm still trying to relearn Haskell. What is this change?

Reply:
Oh... this was something where I deleted the code, replaced it with something else, and then deleted that and wrote it back to its original meaning but apparently not its original state. This is a superfluous change, I can get rid of it.